solutions/src/test/java/ru/sberbank/school/task02/ExConverterTest.java

```diff
@@ -0,0 +1,27 @@
+package ru.sberbank.school.task02;
+
+
+
+import java.math.BigDecimal;
+import java.util.Optional;
+
+import org.junit.jupiter.api.Test;
+
+
+import ru.sberbank.school.task02.util.Beneficiary;
+import ru.sberbank.school.task02.util.ClientOperation;
+import ru.sberbank.school.task02.util.ExternalQuotesServiceDemo;
+import ru.sberbank.school.task02.util.Symbol;
+
+public class ExConverterTest {
+    @Test
+    public void test() {
+        ExternalQuotesService service = new ExternalQuotesServiceDemo();
+        service.getQuotes(Symbol.USD_RUB)
+                .forEach(
+                        quote -> System.out.println(quote.getVolume() + " " + quote.getBid() + " " + quote.getOffer()));
+        ExConverter conv = new ExConverter(service);
+        Optional<BigDecimal> convert = conv.convertReversed(ClientOperation.SELL, Symbol.USD_RUB, BigDecimal.valueOf(137569.4800), Beneficiary.BANK);
```

Comment:
Line is longer than 120 characters (found 149).

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-780338810>*